content/posts/client-mpv.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "Client Spotlight: MPV"
+subtitle = "Watch all your favorite content without transcoding!"
+githubusername = "iwalton3"
```

Comment:
Without an author entry in here (e.g. `author = “Ian Walton”`), then the byline will automatically be set to “The Jellyfin Team”. Do we want that with the iwalton3 avatar?

Reply:
```suggestion
author = “Ian Walton”
githubusername = "iwalton3"
```